README.md

```diff
@@ -6,38 +6,39 @@ This is an open-ended pair lab! It is up to you and your teammate to come up wit
 
 ## Technical Requirements
 
-Your application should be ME(A)N. This includes...
+Your application should be MEAN. This includes...
 * Using **Express** to handle HTTP requests and serve API endpoints.
 * Using **MongoDB** as your database.
 * Using **Mongoose** as your ODM.
 * Adding **Angular** as your front-end. Express should only handle your root view as well as your API routes. Angular should take care of the rest via `ui.router`.
 
```

Comment:
* Using **MongoDB** as your database and **Mongoose** as your ODM
* Using **Express** to handle HTTP requests and serve API endpoints.
* Adding **Angular** as your front-end. Express should only handle your root view as well as your API routes. Angular should take care of the rest via `ui.router`.
* Using **Node** and **Node Package Manager (npm)** to install dependencies and run your ExpressJS server ???

I like that the acronym is laid out fully in the bullets is all...